app/src/main/contrib/components/Responsibilities/ModelRunParameters/ModelRunParametersStatus.tsx

```diff
@@ -19,41 +25,73 @@ export class ModelRunParametersStatus extends React.Component<Props, State> {
         super();
         const storeState = runParametersStore.getState();
         this.state = {
-            sets: storeState.parameterSets
-                ? storeState.parameterSets.filter(s => s.disease == props.disease)
-                : []
+            set: this.getLastSetForDisease(storeState.parameterSets, props.disease),
+            tokens: storeState.oneTimeTokens,
```

Comment:
So much better with only one set in the state - nice one. Can do the same with tokens though, we don't need an array of tokens here, just a single token for the latest set!